flux-middle/src/global_env.rs

```diff
@@ -45,6 +45,7 @@ pub struct GlobalEnv<'sess, 'tcx> {
     adt_variants: RefCell<VariantMap>,
     defns: Defns,
     queries: Queries,
+    extern_fns: RefCell<FxHashMap<DefId, DefId>>,
```

Comment:
I don't think this is ever mutated, so we can drop the `RefCell`

```suggestion
    extern_fns: FxHashMap<DefId, DefId>,
```

Reply:
The `get` requires a mutable borrow unless I'm mistaken. I did have it like that at first.

Reply:
You are mistaken, the entry api does but you shouldn't be using the entry api.